InvasivesBC/ViewControllers/LandingPage/EntryOptionViewController.swift

```diff
@@ -8,12 +8,76 @@
 
 import UIKit
 
+// Enum: EntryOption
+//  Different Data Entry options
+enum EntryOption: String, CaseIterable {
+    case addObservation = "Add Observation"
+    case addMechanicalTreatment = "Add Mechanical Treatment"
+    case addChemicalTreatment = "Add Chemical Treatment"
+    case addMechanicalMonitoring = "Add Mechanical Monitoring"
+    case addChemicalMonitoring = "Add Chemical Monitoring"
+}
+
+// EntryOptionDelegate
+// Delegate to send message of user entry option selection
+protocol EntryOptionDelegate: NSObjectProtocol {
+    func didSelect(entryOption: EntryOption) -> Void
+}
+
+// EntryOptionViewController
+// ViewController to display entry options and manages user selection
```

Comment:
Could have used showOptions() in BaseViewController.
Wrapping use case in that functions allows us to support Mobile more easily if requirements change: Just by using action or popover based on device

Reply:
I will do that but before that, I want to create a navigation system and integrate with all view controllers like prepare segue where two controllers can exchange info.